src/api/rpc_api/author.rs

```diff
@@ -106,13 +106,21 @@ pub trait SubmitAndWatch {
 
 	/// Submit an extrinsic and watch it until the desired status
 	/// is reached, if no error is encountered previously.
+	/// The events are not fetched. So no events are listed in the report.
+	/// To fetch the triggered events, please use submit_and_watch_extrinsic_until.
 	/// Upon success, a report containing the following information is returned:
 	/// - extrinsic hash
 	/// - if watched until at least `InBlock`:
 	///   hash of the block the extrinsic was included in
 	/// - last known extrinsic (transaction) status
+	/// - no events
```

Comment:
Does it make sense to list `no events` in a list where it says ` the following information is returned:` ?

Reply:
In the report we will have `events: None`.  
This should make clear that `events: None` in the report, doesn't mean that no events were triggered, but that the events were not fetched. Perhaps it is not clear enough.  Maybe  "- events: None, because events are not fetched " ?

Reply:
How the report looks like can be looked up easily in the code. So I don't think it's necessary to mention twice (already written above), that event are not fetched

Reply:
Let's try it ;)